cif_pow.dic

```diff
@@ -2158,6 +2158,398 @@ save_pd_calib_d_to_tof.power
+    information that differs by detector channel or ID.
+
+    Common intensity calibration procedures include, but are not limited to:
+     i)  the application of a known, uniform, flood-field; or
+     ii) scanning a detector bank across a peak, or the direct-beam.
+
+    The above examples provide experimental methods to assign values to
+    _pd_calib_intensity.detector_response, to place each detector on a common
+    scale. Note that these are only indicative examples.
+
+    This category is only intended to record detector-related response to
+    a beam incident on the detector. To record variations in intensity during
+    the measurement of a diffractogram, see _pd_meas.intensity_monitor and
+    _pd_meas.counts_monitor.
+;
+    _name.category_id             PD_GROUP
+    _name.object_id               PD_CALIB_DETECTED_INTENSITY
+
+    loop_
+      _category_key.name
+         '_pd_calib_detected_intensity.detector_id'
+         '_pd_calib_detected_intensity.id'
+
+save_
+
+save_pd_calib_detected_intensity.block_id
+
+    _definition.id                '_pd_calib_detected_intensity.block_id'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A block ID code identifying the diffractogram from which the intensity
+    calibration was taken, if it was calibrated by a specimen.
+
+    The data block containing the diffraction pattern will be identified with a
+    _pd_block.id code matching the code in _pd_calib_intensity.block_id.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               block_id
+    _name.linked_item_id          '_pd_block.id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_calib_detected_intensity.detector_id
+
+    _definition.id                '_pd_calib_detected_intensity.detector_id'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A code which identifies the detector or channel number in a
+    position-sensitive, energy-dispersive or other multiple-detector instrument.
+    Note that this code should match the code name used for
+    _pd_meas.detector_id.
+;
+    _name.category_id             pd_calib_detected_intensity
+    _name.object_id               detector_id
+    _name.linked_item_id          '_pd_meas.detector_id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+
+save_
+
+save_pd_calib_detected_intensity.detector_response
+
+    _definition.id
+        '_pd_calib_detected_intensity.detector_response'
+    _definition.update            2023-01-21
+    _description.text
+;
+    A value that indicates the relative sensitivity of each detector. This can
+    compensate for differences in electronics, size and collimation. Usually,
```

Comment:
The definition should probably clarify that response a < response b means that detector a produces less signal than detector b for the same incident illumination, so to bring detectors onto the same scale the observed counts should be divided by the `detector_response`.

Reply:
I've added words to that effect.